ttl/ttlworker/job.go

```diff
@@ -63,14 +82,44 @@ func removeTaskForJob(jobID string) (string, []interface{}) {
 	return removeTaskForJobTemplate, []interface{}{jobID}
 }
 
+func addJobHistorySQL(job *ttlJob, finishTime time.Time, summaryText string) (string, []interface{}) {
+	status := cache.JobStatusFinished
+	if job.status == cache.JobStatusTimeout || job.status == cache.JobStatusCancelled {
+		status = job.status
+	}
+
+	var partitionName interface{}
+	if job.tbl.Partition.L != "" {
```

Comment:
```suggestion
	if job.tbl.Partition.O != "" {
```
This looks better IMO.